spark/src/main/java/org/opensearch/sql/spark/execution/session/SessionState.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.spark.execution.session;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+import lombok.Getter;
+
+@Getter
+public enum SessionState {
+  NOT_STARTED("not_started"),
+  RUNNING("running"),
+  DEAD("dead"),
```

Comment:
Whats the difference between DEAD and FAILED?

Reply:
* CLOSE_SESSION event transit state to DEAD.
* SPARK_APP FINISH SUCCESS event transit state to DEAD.

* SPARK_APP FINISH FAILED event transit state to FAILED